src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts

```diff
@@ -0,0 +1,44 @@
+import { newE2EPage } from "@stencil/core/testing";
+
+import { CSS } from "./resources";
+import { accessible, defaults, hidden, renders } from "../../tests/commonTests";
+
+describe("calcite-shell-center-row", () => {
+  it("renders", async () => renders("calcite-shell-center-row"));
+
+  it("honors hidden attribute", async () => hidden("calcite-shell-center-row"));
+
+  it("defaults", async () =>
+    defaults("calcite-shell-center-row", [
+      {
+        propertyName: "detached",
+        defaultValue: false
+      },
+      {
+        propertyName: "heightScale",
+        defaultValue: "s"
+      },
+      {
+        propertyName: "position",
+        defaultValue: "end"
+      }
+    ]));
+
+  it("should show row content", async () => {
```

Comment:
What is this test for? It doesn't seem to be testing behavior.

Reply:
Heh...I was totes copying what shell-panel did. I can remove it.

Reply:
👌